packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -205,7 +205,9 @@
         pollParticipationEventStatus($selectedProposal?.id).then(() => (statusLoaded = true))
```

Comment:
Await this as well

Reply:
i used  `then` here, as well as with the `updateParticipationOverviewForEventId`, because we dont want to wait for them to be finished

Reply:
Maybe add a comment, so it doesn't get overwritten later :relaxed: